.travis.yml

```diff
@@ -17,3 +18,4 @@ script:
   - npm test
   - cd ..
   - bash ./deploy.sh
+  - curl -O https://sideshowbarker.net/nightlies/jar/vnu.jar && /usr/lib/jvm/java-8-oracle/jre/bin/java -jar vnu.jar --skip-non-html /home/travis/build/whatwg/streams/streams.spec.whatwg.org
```

Comment:
There's not much difference between the install step and the script step. They both run every time. The install step just has a default depending on your language (which was overriden in the original).